server/templates/student/course/_assigntable.html

```diff
@@ -9,7 +9,7 @@
           {% if scores %}<th class="col-scores">Scores</th> {% endif %}
           <th class="col-status">Status</th>
           <th class="col-due-date">Due Date</th>
-          <th class="col-group">Group</th>
+          <th class="col-group hidden-md">Group</th>
```

Comment:
Can we keep group in for now?  I don't think it's ok to hide information at a specific screen size. 

It seems like we have plenty of room on the top 
![image](https://cloud.githubusercontent.com/assets/882381/17955711/f8c2136a-6a38-11e6-9f42-0464f63ddc66.png)